huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_instance.go

```diff
@@ -146,7 +145,14 @@ func ResourceDmsRocketMQInstance() *schema.Resource {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Computed:    true,
-				Description: `Specifies the ACL access control.`,
+				Description: `Specifies whether access control is enabled.`,
+				Deprecated:  "Use 'enable_acl' instead",
```

Comment:
please put the Deprecated argument in the end of schema.

Reply:
done